tasks/sync-granule/index.js

```diff
@@ -126,7 +128,12 @@ function syncGranule(event) {
     // eslint-disable-next-line camelcase
     const granuleDuplicates = {};
     const granules = [];
+    const createdAt = workflowStartTime || Date.now();
```

Comment:
I think the `|| Date.now()` is unnecessary based on the logic above, especially with the suggested refactor

Reply:
Agreed.